composer.lock

```diff
@@ -1,1764 +0,0 @@
-{
```

Comment:
`composer.lock` is not needed in libraries - it just makes it harder to test compatibility with a variety of different PHP/PhpUnit versions